app/setting/controllers/setting-users-controller.js

```diff
@@ -64,6 +66,10 @@ function (
         return false;
     };
 
+    $scope.addUser = function () {
+        $location.path('/settings/users/create');
```

Comment:
I *think* it actually already works. Using `class="button button-secondary"` on an `<a>` get the right visual